static/webpack-config.js

```diff
@@ -127,11 +127,42 @@ module.exports = function () {
             }
           ],
         },
+        {
+          test: /\.html$/i,
+          use: [
+            {
+              loader: 'html-loader',
+              options: {
+                sources: {
+                  list: [
+                    {
+                      tag: 'link',
+                      attribute: 'href',
+                      type: 'src',
+                      filter: (tag, attribute, attributes) => {
+                        const isPreload = attributes.find(attr => {
+                          return attr.name === 'rel' && attr.value === 'preload';
+                        });
+                        const isFont = attributes.find(attr => {
+                          return attr.name === 'as' && attr.value === 'font';
+                        });
+                        return isPreload && isFont;
```

Comment:
Does this filter mean that only preload fonts are updated to use the hash in their name? Do we not need this for other assets/

Reply:
Fonts are the only assets that still run through webpack, because we import them in our css using resolve-url-loader. We could update the css to not use resolve-url-loader, though I'm a little worried about that breaking people's custom fonts